frontend/src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx

```diff
@@ -144,30 +145,40 @@ export const FeatureToggleListTable: VFC = () => {
                     width: '50%',
                 },
             }),
-            columnHelper.accessor((row) => row.segments?.join('\n') || '', {
-                header: 'Segments',
-                cell: ({ getValue, row }) => (
-                    <FeatureSegmentCell value={getValue()} row={row} />
-                ),
-                enableSorting: false,
-                meta: {
-                    width: '1%',
-                },
-            }),
-            columnHelper.accessor(
-                (row) =>
-                    row.tags
-                        ?.map(({ type, value }) => `${type}:${value}`)
-                        .join('\n') || '',
-                {
-                    header: 'Tags',
-                    cell: FeatureTagCell,
-                    enableSorting: false,
-                    meta: {
-                        width: '1%',
-                    },
-                },
-            ),
+            ...(!flagsReleaseManagementUI
+                ? [
+                      columnHelper.accessor(
+                          (row) => row.segments?.join('\n') || '',
+                          {
+                              header: 'Segments',
+                              cell: ({ getValue, row }) => (
+                                  <FeatureSegmentCell
+                                      value={getValue()}
+                                      row={row}
+                                  />
+                              ),
+                              enableSorting: false,
+                              meta: {
+                                  width: '1%',
+                              },
+                          },
+                      ),
+                      columnHelper.accessor(
+                          (row) =>
```

Comment:
can we write explicit type so that the next person doesn't have to check the data structure at runtime?